examples/model_selection/plot_grid_search_refit_callable.py

```diff
@@ -0,0 +1,116 @@
+"""
+==================================================
+Balance model complexity and cross-validated score
+==================================================
+
+This example balances model complexity and cross-validated score by
+finding a decent accuracy within 1 standard deviation of the best accuracy
+score while minimising the number of PCA components. This is a rule of thumb
```

Comment:
The rule of thumb has nothing to do with PCA components, so this text is unclear.